src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -282,63 +286,54 @@ void func_80A85E2C(EnTruUnkStruct* arg0, Vec3f* arg1, Vec3f* arg2, Vec3f* arg3,
-                alpha = (f32)arg0->unk_02 / arg0->unk_01;
-                alpha *= D_80A8B274[arg0->unk_00 - 3];
-            } while (0);
+        Matrix_Push();
 
-            gDPSetCombineLERP(POLY_XLU_DISP++, PRIMITIVE, ENVIRONMENT, TEXEL0, ENVIRONMENT, PRIMITIVE, 0, TEXEL0, 0,
-                              COMBINED, 0, SHADE, 0, 0, 0, 0, COMBINED);
-            gDPSetRenderMode(POLY_XLU_DISP++, G_RM_FOG_SHADE_A, G_RM_ZB_CLD_SURF2);
-            gSPSetGeometryMode(POLY_XLU_DISP++, G_FOG | G_LIGHTING);
-            gDPPipeSync(POLY_XLU_DISP++);
-            gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, D_80A8B25C[arg0->unk_00 - 3].r, D_80A8B25C[arg0->unk_00 - 3].g,
-                            D_80A8B25C[arg0->unk_00 - 3].b, (u8)alpha);
-            gDPSetEnvColor(POLY_XLU_DISP++, D_80A8B25C[arg0->unk_00 - 3].r, D_80A8B25C[arg0->unk_00 - 3].g,
-                           D_80A8B25C[arg0->unk_00 - 3].b, 0);
+        alpha = (f32)arg0->unk_02 / arg0->unk_01;
+        alpha *= D_80A8B274[arg0->unk_00 - 3];
 
-            Matrix_Translate(arg0->unk_04.x, arg0->unk_04.y, arg0->unk_04.z, MTXMODE_NEW);
-            Matrix_Scale(arg0->unk_28, arg0->unk_28, 1.0f, MTXMODE_APPLY);
-            Matrix_ReplaceRotation(&play->billboardMtxF);
+        gDPSetCombineLERP(POLY_XLU_DISP++, PRIMITIVE, ENVIRONMENT, TEXEL0, ENVIRONMENT, PRIMITIVE, 0, TEXEL0, 0,
+                          COMBINED, 0, SHADE, 0, 0, 0, 0, COMBINED);
+        gDPSetRenderMode(POLY_XLU_DISP++, G_RM_FOG_SHADE_A, G_RM_ZB_CLD_SURF2);
+        gSPSetGeometryMode(POLY_XLU_DISP++, G_FOG | G_LIGHTING);
+        gDPPipeSync(POLY_XLU_DISP++);
+        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, D_80A8B25C[arg0->unk_00 - 3].r, D_80A8B25C[arg0->unk_00 - 3].g,
+                        D_80A8B25C[arg0->unk_00 - 3].b, (u8)alpha);
+        gDPSetEnvColor(POLY_XLU_DISP++, D_80A8B268[arg0->unk_00 - 3].r, D_80A8B268[arg0->unk_00 - 3].g,
+                       D_80A8B268[arg0->unk_00 - 3].b, 0);
 
-            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-            index = ((f32)arg0->unk_02 / arg0->unk_01) * 8.0f;
-            gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[index]));
-            gSPDisplayList(POLY_XLU_DISP++, gKoumeDustModelDL);
+        Matrix_Translate(arg0->unk_04.x, arg0->unk_04.y, arg0->unk_04.z, MTXMODE_NEW);
+        Matrix_Scale(arg0->unk_28, arg0->unk_28, 1.0f, MTXMODE_APPLY);
+        Matrix_ReplaceRotation(&play->billboardMtxF);
 
-            Matrix_Pop();
-        }
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        index = ((f32)arg0->unk_02 / arg0->unk_01) * 8.0f;
+        gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[index]));
+        gSPDisplayList(POLY_XLU_DISP++, gKoumeDustModelDL);
+        Matrix_Pop();
     }
-
-    CLOSE_DISPS(play->state.gfxCtx);
+    CLOSE_DISPS(new_var);
```

Comment:
This only works because `CLOSE_DISPS` just evaluates to a closed bracket in retail, but revert this to `CLOSE_DISPS(play->state.gfxCtx);` since `new_var` isn't a variable that exists in scope